tests/CMakeLists.txt

```diff
@@ -79,6 +79,7 @@ if(WITH_PYTHON)
     add_fdb_test(TEST_FILES SimpleExternalTest.txt)
   else()
     message(WARNING "Python not found, won't configure ctest")
```

Comment:
I think this message "python not found" doesn't make sense here, would you mind removing it?